spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AppRegistryController.java

```diff
@@ -391,12 +392,11 @@ public void unregisterAll() {
 	 * Register all applications listed in a properties file or provided as key/value pairs.
 	 *
 	 * @param pageable Pagination information
-	 * @param pagedResourcesAssembler the resource asembly for app registrations
+	 * @param pagedResourcesAssembler the resource assembly for app registrations
 	 * @param uri URI for the properties file
 	 * @param apps key/value pairs representing applications, separated by newlines
 	 * @param force if {@code true}, overwrites any pre-existing registrations
 	 * @return the collection of registered applications
-	 * @throws IOException if can't store the Properties object to byte output stream
```

Comment:
Shouldn't the API throw ApplicationException rather than an IOException?

Reply:
This was a different method that wasn't throwing the exception, so I removed it.